packages/search-in-workspace/src/browser/styles/index.css

```diff
@@ -0,0 +1,315 @@
+.searchContainer {
+    color: var(--theia-ui-font-color1);
+    padding: 5px;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    box-sizing: border-box;
+}
+
+.searchContainer input[type="text"] {
+    flex: 1;
+    line-height: var(--theia-content-line-height);
+    font-size: var(--theia-ui-font-size1);
+    padding-left: 8px;
+    border-width: 1px;
+    border-style: solid;
+    border-color: var(--theia-border-color1);
+    background: var(--theia-layout-color0);
+    color: var(--theia-ui-font-color1);
+}
+
+.searchContainer input[type="text"]:focus {
+    outline: none;
+}
+
+.searchContainer .searchHeader {
+    width: 100%;
+    margin-bottom:  10px;
+}
+
+.searchContainer .searchHeader .controls.button-container {
+    height:30px;
+    margin-bottom: 5px;
+}
+
+.searchContainer .searchHeader .controls .refresh {
+    background: var(--theia-icon-refresh);
+}
+
+.searchContainer .searchHeader .controls .collapse-all {
+    background: var(--theia-icon-collapse-all);
+}
+
+.searchContainer .searchHeader .controls .clear {
+    background: var(--theia-icon-clear);
+}
+
+.searchContainer .searchHeader .search-field {
```

Comment:
please use more specific top level css class names to avoid collisions: `theia-search-in-workspace-container`

@kittaakos was suggesting to use `t-` prefix instead to make it shorter: `t-siw-container`

We can discuss css convention on next dev meeting, agree on something as a team and follow it.

Reply:
done